plan/logical_plan_test.go

```diff
@@ -612,19 +612,19 @@ func (s *testPlanSuite) TestPlanBuilder(c *C) {
 		},
 		{
 			sql:  "update t set t.a = t.a * 1.5 where t.a >= 1000 order by t.a desc limit 10",
-			plan: "DataScan(t)->Sel([ge(test.t.a, 1000)])->Sort->Limit->*plan.Update",
+			plan: "TableReader(Table(t)->Limit)->Limit->Update",
```

Comment:
Why DataScan is replaced with TableReader?

Reply:
We build update's `SelectPlan` at one time.